sandy/core/endf6.py

```diff
@@ -1677,33 +1667,54 @@ def get_errorr(self,
             neutron group option (default is 2, csewg 239-group structure)
         ek : iterable, optional
             derived cross section energy bounds (default is `[1e-5, 2e7]`)
-        iwt : `int`, optional
-            weight function option (default is 2, constant)
         sigz : iterable of `float`
             sigma zero values. The default is 1.0e10.
+        iwt_groupr : `int`, optional
+            weight function option (default is 2, constant)
+        spect_groupr : iterable, optional
+            Weight function as a iterable (default is None)
```

Comment:
i would rather pass it here as a series and then intertwine the data into a single array. Can this be called `spectrum_groupr`? there is no need for abbreviations